sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -57,20 +57,20 @@ export class InstitutionLocationsController extends BaseController {
     }
 
     //To retrive institution id
-    const institutionDetails = await this.institutionService.getInstituteByUserName(
-      userToken.userName,
-    );
+    const institutionDetails =
```

Comment:
Just checking, is there any change here related to the PR or just the formatting?

Reply:
ya, it's just formatting